docs/section19/Rule 19-1.md

```diff
@@ -0,0 +1,52 @@
+# Section 19 - Rule 19-1  
+**Schema Version:** 0.0.21  
+**Mandatory Rule:** True  
+**Rule ID:** 19-1   
+**Rule Description:** HVAC system coil capacities for the baseline building design shall be oversized by 15% for cooling and 25% for heating..  
+**Rule Assertion:** Options are Pass/Fail/IN_APPLICABLE     
+**Appendix G Section:** Section G3.1.2.2    
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Each HeatingVentilationAirConditioningSystem Data Group  
+
+**Applicability Checks:** 
+
+1. Applies to the following baseline system types: 1, 1b, 2, 3, 3a, 3b, 4, 5, 5b, 6, 6b, 9 (systems with furnaces, heat pumps, and DX cooling coils)  
+
+**Function Calls:**  
+1. get_baseline_system_types()  
+
+
+## Rule Logic:  
+**Applicability Check 1:**  At the whole building level
+- Get B-RMR system types: `baseline_hvac_system_dict = get_baseline_system_types(B-RMR)`  
+- Check if this rule is applicable to any HVAC systems in the B_RMR, if not then the outcome is IN_APPLICABLE for all HVAC systems: `If any(hvac.id in baseline_hvac_system_dict[sys_type] for sys_type in ["SYS-1", "SYS-1b","SYS-2", "SYS-3", "SYS-3a", "SYS-4", "SYS-5", "SYS-5b", "SYS-6", "SYS-6b", "SYS-9"]):`   
+    - For each hvac system in B_RMR: `for hvac in B_RMR...HeatingVentilationAirConditioningSystem:`    
+        - Reset heating_oversizing_factor: `heating_oversizing_factor = ""`  
+        - Reset cooling_oversizing_factor: `cooling_oversizing_factor = ""`  
+        - Get baseline system type: `sys_type = list(baseline_hvac_system_dict.keys())[list(baseline_hvac_system_dict.values()).index(hvac.id)]`  
+        **Applicability Check 2:** At the component level
+        - Check if the baseline system type has a furnace, heat pump, or DX cooling coil, ELSE then rule outcome for this HVAC system is NOT_APPLICABLE: `if sys_type in ["SYS-1", "SYS-1b","SYS-2", "SYS-3", "SYS-3a", "SYS-4", "SYS-5", "SYS-5b", "SYS-6", "SYS-6b", "SYS-9"]:`  
```

Comment:
@claperle for checks like these where we're identifying whether heating system or cooling system is a certain type, is there value in simply calling the sub-function for is cooling DX or is heating furnace rather than the system type function? i do not know if one works better than the other

Reply:
@supriyagoel This is a good point. I'll think about this going forward as I am not sure which is better in terms of ease of creating the RDS and code efficiency at the moment. Are you on board with leaving this one as is?